libavcodec/gif.c

```diff
@@ -125,6 +125,8 @@ static void gif_crop_translucent(AVCodecContext *avctx,
     GIFContext *s = avctx->priv_data;
     int trans = s->transparent_index;
 
+    av_log(avctx, AV_LOG_DEBUG,"avctx frame: %d\n", avctx->frame_number);
```

Comment:
let's get rid of all of these extra logs so we can maybe prep a patch for upstream